### PR TITLE
Update Chinese translations

### PR DESCRIPTION
**Pull request by CookiePieWw**

### CODE REVIEW

File: po/zh.po [hunk 1]
```diff
@@ -151,76 +151,85 @@ msgstr "缓存文件"
...
 "Newsboat is free software licensed under the MIT License. (Type `%s -vv' to "
 "see the full text.)"
 msgstr ""
+"Newsboat 是一个自由软件，根据 MIT 许可证授权。（输入 `%s -vv' 以"
+"查看许可证全文。）"
 
 #: newsboat.cpp:178
 msgid "It bundles:"
-msgstr ""
+msgstr "它使用了"
```
Comment (der-lyse) [inline po/zh.po:172]: A colon is probably missing here at the end.

File: po/zh.po [hunk 2]
```diff
@@ -71,58 +71,58 @@ msgstr "从<配置文件>里读取配置信息"
...
 msgid "get version information"
 msgstr "获得版本信息"
 
 #: newsboat.cpp:81 src/pbcontroller.cpp:348
 msgid "<loglevel>"
-msgstr "记录等级"
+msgstr "<日志等级>"
 
 #: newsboat.cpp:82
 #, fuzzy
```
Comment (der-lyse) [inline po/zh.po:97]: Since you addressed this translation, you can remove this fuzzy marker and all the other ones.

File: po/zh.po [hunk 3]
```diff
@@ -1047,12 +1059,12 @@ msgstr "没有未读的文章"
...
 #: src/view.cpp:915
 msgid "Already on last item."
-msgstr ""
+msgstr "已经位于最后一项。"
 
 #: src/itemlistformaction.cpp:447 src/itemviewformaction.cpp:380
 #: src/view.cpp:881
 msgid "Already on first item."
-msgstr ""
+msgstr "已经位于第一项"
```
Comment (der-lyse) [inline po/zh.po:1067]: A full stop is probably missing at the end. The translation immediately before this here uses one, however, the one above that doesn't either. So perhaps you could make this somewhat consistent either way, whatever suits the Chinese language best.

Btw I like how Chinese has its own typographic parentheses, exclamation marks and full stops. :-)

File: po/zh.po [hunk 4]
```diff
@@ -1101,7 +1113,7 @@ msgstr[0] ""
 #: src/itemlistformaction.cpp:1377 src/itemviewformaction.cpp:229
 #: src/itemviewformaction.cpp:579
 msgid "Aborted saving."
-msgstr "放弃保存"
+msgstr "保存中断"
```
Comment (der-lyse) [inline po/zh.po:1116]: Same here, a full stop might be missing at the end.

**Review by der-lyse — COMMENTED** [review]
Thank you! I know nothing about Chinese, but I tried to focus on general stuff like punctuation, usage of keys and such.

In case my little findings are actually valid, it's up to you when you want to address them. Ideally you repair them before we merge this. But we can also proceed and merge it right away (I'd just ask you to remove the now obsolete "fuzzy" markers) and you fix them with another MR at a later date.

**Comment by der-lyse** [comment]
Oh, you also might want to bump the `PO-Revision-Date` and include yourself in the `Last-Translator` and `Language-Team` fields in lines 6-8. :-)

**Comment by CookiePieWw** [comment]
@der-lyse Thanks for your advice! I'm working on it, but not sure about something here.

> Oh, you also might want to bump the `PO-Revision-Date` and include yourself in the `Last-Translator` and `Language-Team` fields in lines 6-8. :-)

For the `PO-Revision-Date`, since I'm not using a po-editor tool, is it appropriate to choose a time around and just fill it manually?

For the `Last-Translator` and `Language-Team`, is it alright to move `josh yu`(which has been there before) down to the language team and fill in mine?

**Comment by der-lyse** [comment]
Cool, @CookiePieWw!

> For the `PO-Revision-Date`, since I'm not using a po-editor tool, is it appropriate to choose a time around and just fill it manually?

Absolutely, that's exactly how I do it, too. :-) With releases every few months, in my opinion only the date part matters, so we can tell how recent or old certain translations are. The time itself doesn't add a lot of value in our case. I just noticed in the past that e.g. `make run-i18nspector` warns when using a timestamp in the future. 

> For the `Last-Translator` and `Language-Team`, is it alright to move `josh yu`(which has been there before) down to the language team and fill in mine?

Exactly, you would be the last translator and the language team consists of both of you. I think new translators just add themselves at the end of this comma-separated language team list.

By the way, would you like to be added to the _Newsboat Translators_ team here on GitHub? That way, you'll get notified two weeks before the next release, so you can go ahead and update the translations? :-)

**Comment by CookiePieWw** [comment]
@der-lyse Got it. I'll make some changes later in this PR.

> By the way, would you like to be added to the _Newsboat Translators_ team here on GitHub? That way, you'll get notified two weeks before the next release, so you can go ahead and update the translations? :-)

Sure I'd like to. It's such a good software and I'm looking forward to releases in the future.

**Comment by der-lyse** [comment]
Awesome, looking forward to your changes! Apparently, I cannot add new team members, but @Minoru can for sure. ;-)

**Comment by CookiePieWw** [comment]
There are still a lot unresolved items and I'm afraid I cannot take them all down at once in this PR. I think I will take a look at times.

**Review by der-lyse — APPROVED** [review]
This is cool, thank you very much! Continuously improving the translations is a good approach.

**Pull request merged**